src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1181,10 +1186,14 @@ void RaftReplDev::on_log_found(logstore_seq_num_t lsn, log_buffer buf, void* ctx
     }
 
     rreq->set_lsn(repl_lsn);
+    // keep lentry in scope for the lyfe cycle of the rreq
```

Comment:
I probably didn't get why we need to keep this lentry in rreq. Isn't everything copied and saved in rreq, or is there a place that is referring to lentry until rreq finishes?

Reply:
homeobject accesses the header during commit which is formed from the lentry